src/mlpack/tests/io_test.cpp

```diff
@@ -833,7 +838,8 @@ TEST_CASE_METHOD(IOTestDestroyer, "OutputMatrixNoTransposeParamTest",
 
   // Now load the matrix back and make sure it was saved correctly.
   arma::mat dataset2;
-  data::Load("test.csv", dataset2, true, false);
+  if (!data::Load("test.csv", dataset2, true, false)) 
```

Comment:
```suggestion
  if (!data::Load("test.csv", dataset2, false, false)) 
```
Third argument should be changed to 'false'.